teams/project-vision-doc-2025.toml

```
@@ -0,0 +1,25 @@
+name = "project-vision-doc-2025"
+kind = "project-group"
+subteam-of = "leadership-council"
```

I think this should probably be:

```suggestion
subteam-of = "launching-pad"
```

We don't want any teams (other than the top-level teams) directly linked under the council. I think as we move towards having the launching pad (name TBD) be a home for cross-cutting teams, this somewhat fits. Or, we find some different structure in the future, in which case we can move things.